Research/rohanhh2/flutter_practice_pt3

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import 'package:random_string_generator/random_string_generator.dart';
```

Comment:
You can find the documentation for this package at: https://pub.dev/packages/random_string_generator